velox/connectors/hive/iceberg/FilterUtil.cpp

```diff
@@ -20,6 +20,21 @@ namespace facebook::velox::connector::hive::iceberg {
 using namespace facebook::velox::exec;
 using namespace facebook::velox::core;
 
+std::unique_ptr<common::Filter> createNegatedBytesValuesFilter(
+    const VectorPtr& valuesVector,
+    vector_size_t offset,
+    vector_size_t size) {
+  auto [values, hasNull] =
+      facebook::velox::common::deDuplicateValues<std::string, StringView>(
+          valuesVector, offset, size);
+
+  VELOX_USER_CHECK(
+      !values.empty() || hasNull,
```

Comment:
Why is this check required? Its not present in createNegatedBigintValuesFilter. If the values is empty and hasNull is true, we still create the filter which goes against the error message here. If it hasNull, then shouldn't check the values vector size is greater than one?